src/Layouts/Partials/FillItem.vala

```diff
@@ -166,6 +168,14 @@ public class Akira.Layouts.Partials.FillItem : Gtk.Grid {
         fill_chooser.attach (color_container, 1, 0, 1, 1);
         fill_chooser.attach (opacity_container, 2, 0, 1, 1);
 
+        eye_dropper_button = new Gtk.Button ();
+        eye_dropper_button.get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
+        eye_dropper_button.get_style_context ().add_class ("button-rounded");
+        eye_dropper_button.can_focus = false;
+        eye_dropper_button.valign = Gtk.Align.CENTER;
+        eye_dropper_button.add (new Gtk.Image.from_icon_name ("user-trash-symbolic",
+            Gtk.IconSize.SMALL_TOOLBAR));
```

Comment:
Let's add a `tooltip_text` to this button that reads "Pick color".
I'll need to add tooltips for all the other icons.

Reply:
Yah I thought about it, I was thinking on making a PR adding a tooltip on everything there. But no prob, I'll